src/components/PictureCard.jsx

```diff
@@ -0,0 +1,162 @@
+import styled from "styled-components";
+
+import { eventButtonVector } from "../images";
+
+import { Button } from "./styleElements";
+
+import { BreakPoints } from "../components";
+
+import useCounter from "./hooks/useDonationLogic";
+
+import PropTypes from "prop-types";
```

Comment:
wydaje mi sie ze można wyłączyć te bugi z prop-types gdzieś w konfiguracji, ale to na marginesie sugestia